src/utilities/processo-juridico/index.ts

```diff
@@ -21,3 +21,34 @@ export function format(processoJuridico: string) {
       return result;
     }, '');
 }
+
+export function verifyDigit(processo: string): boolean {
+  const digits = processo.split('');
+  const verificationDigits = digits.splice(7, 2).join('');
+
+  const digits1to11 = digits.slice(0, 11).reduce((acc, digit, index) => {
+    return acc + +digit * Math.pow(10, 10 - index);
+  }, 0);
+
+  const firstRemainder = digits1to11 % 97;
```

Comment:
Maybe you could put `97` in a variable to give some description of what it is.

Reply:
Great suggestion, I added all these numbers into variables and also added a brief comment describing the MOD97_10 formula.